src/trajectory.jl

```diff
@@ -211,7 +213,7 @@ nsteps(τ::Trajectory{TS, I, TC}) where {TS, I, TC<:FixedIntegrationTime} =
 ## Kernel interface
 ##
 
-struct HMCKernel{R, T<:Trajectory} <: AbstractMCMCKernel 
+struct HMCKernel{R, T<:Trajectory} <: AbstractMCMCKernel
```

Comment:
Just to double check this is just the removal of a space?

Reply:
Yep. But also, now `AbstractMCMCKernel` is `AbstractMCMC.AbstractSampler` (I'll remove this alias, and make it explicit).

Reply:
Is there any side effect it might cause?

Reply:
Don't think so. Anything particular in mind or just general question? 

The only thing is that if we remove `AbstractMCMCKernel` completely, not even aliasing, e.g. CoupledHMC.jl won't work.

Maybe best approach is to just make `AbstractMCMCKernel <: AbstractMCMC.AbstractSampler`?

Reply:
I think it's fine as long as CoupledHMC is fine (with sutff like mixture kernels still working).